src/cast_core/src/CollectionScanner.cpp

```diff
@@ -508,11 +507,13 @@ void CollectionScanner::run() {
 
 CollectionScanner::CollectionScanner(genny::ActorContext& context)
     : Actor{context},
+      // TODO: _totalInserts is unused
```

Comment:
Do you want a ticket number for the TODO (same question for line 516)

Reply:
In reality I don't suspect that a ticket would ever get prioritized so would just be noise. This Actor is owned by Storage Engines. I think this comment is only useful if/when this Actor gets any attention from future maintainers; this change isn't all that important on its own.